api/.project/devstart.rb

```diff
@@ -0,0 +1,51 @@
+require_relative "common/common"
+
+def startdb()
```

Comment:
Crap. I want these commands to be self-documenting. Let me explain a couple of things and see if that changes your mind. First, you probably were expecting this to do more, but I haven't added the database creation scripts yet, so you didn't miss those—they aren't there. Second, if you type `docker run mysql` and you don't have that image, docker automatically pulls it. This is something you get used to quickly. The `%W{}` just says, "split these things by words" (fairly idiomatic ruby) and `#{}` is ruby's string substitution mechanism. So that whole thing is basically just one `docker run` command with a `docker rm` to remove the container when the script exits. So, if you look with fresh eyes, does it still need a docstring?